### PR TITLE
updates to README file

### PR DESCRIPTION
**Pull request by beav**
This freshens some stale info in the README and fixes a broken link.

### CODE REVIEW

**Comment by stbenjam** [comment]
@beav If you go here: https://github.com/orgs/Katello/people?utf8=%E2%9C%93&query=beav, you can make your membership in the org public and you'll automatically get your PR's in Katello and Foreman orgs tested.

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Linux based systems.
 
 ## Development
 
-The recommended way to set up Katello for development to use the [katello-devel-installer](https://github.com/Katello/katello-installer#development-usage) directly or via [katello-deploy](https://github.com/Katello/katello-deploy#development-deployment).
+The recommended way to set up Katello for development to use the [katello-devel-installer](https://github.com/Katello/katello-installer#development-usage) directly or via [katello-deploy](https://github.com/Katello/katello-deploy#development-deployment) if you prefer to use Vagrant. If you run into issues during the development setup, feel free to ask for help on #theforeman-dev or via the foreman-dev mailing list.
```
Comment (stbenjam) [inline README.md]: katello-deploy can actually do non-vagrant too, ./setup.rb works on any centos 6/7 box

File: README.md [hunk 2]
```diff
@@ -29,14 +29,13 @@ At this point, the development environment should be completely setup and the Ka
     rails s
     ```
 
-2. Access Foreman in your browser (e.g. `http://<hostname>:3000/`)
-3. Login to Foreman (default: `admin` and `changeme`)
-4. Create an initial Foreman organization
-5. Navigate to the Katello engine (e.g. `http://<hostname>:3000/katello`)
+1. Access Foreman in your browser (e.g. `http://<hostname>:3000/`)
+1. Login to Foreman (default: `admin` and `changeme`)
+1. If you go to `http://<hostname>:3000/about` and view the "Plugins" tab, you should see a "Katello" plugin listed.
```
Comment (stbenjam) [inline README.md]: Would you mind changing these URL's to `https://<hostname>`? The katello-devel installer actually sets up an apache reverse proxy to :3000.

File: README.md (hunk 1 above)
Comment (beav) [inline README.md]: Should I change the doc to only mention the two styles of running katello-deploy, and remove the reference to katello-devel-installer?

**Comment by beav** [comment]
@stbenjam thanks for the heads up on membership, I just changed my status

File: README.md (hunk 1 above)
Comment (beav) [inline README.md]: oh n/m, I see what you mean.

File: README.md [hunk 3]
```diff
@@ -29,14 +41,13 @@ At this point, the development environment should be completely setup and the Ka
     rails s
     ```
 
-2. Access Foreman in your browser (e.g. `http://<hostname>:3000/`)
-3. Login to Foreman (default: `admin` and `changeme`)
-4. Create an initial Foreman organization
-5. Navigate to the Katello engine (e.g. `http://<hostname>:3000/katello`)
+1. Access Foreman in your browser (e.g. `https://<hostname>:3000/`)
```
Comment (stbenjam) [inline README.md]: Should be without `:3000`, the apache proxy is on just normal port 443.

**Comment by stbenjam** [comment]
Otherwise looks good :+1:

**Comment by stbenjam** [comment]
Thanks @beav!

**Pull request merged**